CHANGELOG.txt

```diff
@@ -2,6 +2,13 @@
 MABWiser CHANGELOG
 =====================
 
+
+-------------------------------------------------------------------------------
+December, 17, 2019 1.7.1
+-------------------------------------------------------------------------------
+minor:
+- Bug fix for partial fitting in UCB1
```

Comment:
in here, can we acknowledge/thank @xx for bringing this bug to our attention

Reply:
Sure!